src/Components/Links.js

```diff
@@ -8,6 +8,7 @@ function Links({ links }) {
     youtube: 'red',
     twitter: 'blue',
     github: 'black',
+    globe: 'grey',
```

Comment:
what is globe?

Reply:
Hi @eddiejaoude Its an icon on the prime react docs. When rendering the links I noticed there wasn't an option for website links.

#### screenshot
![globe icon eddiehub](https://user-images.githubusercontent.com/50571688/133115435-b1f8b379-e01f-44a6-ab9b-16cc772f2969.jpg)

Reply:
I would change it to `website`

Reply:
Ah ok, thank you. Next time I would recommend added a screenshot to the PR it will make things clearer